lib/couchbaseX.js

```diff
@@ -333,12 +337,7 @@ class CouchbaseAccessor extends Accessor {
    * Result is a promise with the data or an error.
    */
   findById(id, options) {
-    return this.connection.call('getAsync', id, options).then(this.unpackRes).then((data) => {
-      if (data == null) {
```

Comment:
Is this a break change? Previously `findById` throw 404 error when the doc for Id is missing?

Reply:
no, previously it never reached here.